.readthedocs.yml

```diff
@@ -3,5 +3,7 @@ build:
 
```

Comment:
The `version` top level key [is marked as required on the `readthedocs` documentation](https://docs.readthedocs.io/en/stable/config-file/v2.html#version).

> If you don’t provide the version, v1 will be used.

I think we should add a line specifying `version: 2` to this yaml file.

Reply:
Done. Thanks!